manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
+PackageVersion: 3.0.0
+PackageLocale: en-US
+Publisher: Totally Useful Software, Inc.
+PackageName: QuickSFV
+License: Unknown
+ShortDescription: QuickSFV

```

Comment:
```
ShortDescription: QuickSFV
```
Please write a better description for what this application is about. You can grab these from the publisher's website.

Reply:
Thats what im going to do from now on, didn't think about doing that until you mentioned it